### PR TITLE
Docs: Privacy and security update

### PR DESCRIPTION
**Pull request by Jermolene**
A new badge for the home page to advertise a new section giving basic information about privacy and security when using TiddlyWiki. The goal is to give users good quality pointers to trusted sources of information, and suggest strategies for using TiddlyWiki with minimal trust in external entities.

image

### CODE REVIEW

**Comment by xcazin** [comment]
Hi @Jermolene I believe the first paragraph "Basic Principles" should actually be the last one, something like "Going farther", so that the section starts with material about TiddlyWiki, not EFF.

**Comment by ericshulman** [comment]
> Because data is stoerd in simple text files, people today choosing to use [TiddlyWiki](https://deploy-preview-8746--tiddlywiki-previews.netlify.app/#TiddlyWiki) can be confident that their data will still be accessible in the decades to come.

typo:
"stoerd"

grammar:
change "people today choosing to use TiddlyWiki" to "people choosing to use TiddlyWiki today"

I also agree with @xcazin regarding the positioning of the paragraph about EFF (it should come at the end, not the beginning), perhaps with a heading of "More information"

**Comment by ericshulman** [comment]
Regarding the order of the "badges" in "Find Out More", I suggest:

1. Twenty Years of TiddlyWiki
2. Introduction to TiddlyWiki
3. Your Privacy and Security
4. What's New in vX.Y.Z
5. TiddlyWiki Newsletter
6. Grok TiddlyWiki (omit the version #... it can be mistakenly interpreted as referring to "TiddlyWiki 2.0")
7. links.tiddlywiki.org (this could have a better heading, perhaps "Community Links")
8. How is Tiddlywiki Funded?
9. TiddlyWiki Marketplace
10. Intertwingled Innovations

**Comment by pmario** [comment]
The [TW Privacy and Security](https://deploy-preview-8746--tiddlywiki-previews.netlify.app/#TiddlyWiki%20Privacy%20and%20Security) tiddler contains 13 TiddlyWiki links - I think one at the beginning and one at the end would be enough.

File: editions/tw5.com/tiddlers/workingwithtw/TiddlyWiki Privacy and Security.tid [hunk 1]
```diff
@@ -27,7 +27,7 @@ Users can employ the same tools that they use to keep any document and photograp
 
 There are also open source alternatives such as [[Syncthing|https://syncthing.net/]] that allow users to synchronise files between their devices without using a third party service.
 
-Users may wish to use additional layers of security. TiddlyWiki in the single file configuration itself offers built-in encryption. It uses an industry standard encryption library to offer AES 128 bit encryption in CCM mode. All the data within the file is encrypted, and cannot be accessed without entering the correct password. The password is never stored in the file, and so if it is lost, the data is lost.
+Users may wish to use additional layers of security. TiddlyWiki in the single file configuration itself offers built-in [[encryption|Encryption]]. It uses an industry standard encryption library to offer AES 128 bit encryption in CCM mode. All the data within the file is encrypted, and cannot be accessed without entering the correct password. The password is never stored in the file, and so if it is lost, the data is lost. Instructions can be found [[here|Encryption]]
```
Comment (pmario) [inline editions/tw5.com/tiddlers/workingwithtw/TiddlyWiki Privacy and Security.tid:30]: IMO using "here" as a link is sub-optimal. I would like to  "Instructions can be found at: [[Encryption]]" -- IMO the first "encryption" link could be removed then.

**Comment by Jermolene** [comment]
Thanks @xcazin @ericshulman I've made a further update in 0ea567b86df32b7106ce815f09497287a534ce13

> I don't see any "Developer" badge, only the 10 badges I previously listed.

My mistake, it was listed in the tag list, leftover from when there was a "Developer" badge. I removed it in the above commit.

**Comment by ericshulman** [comment]
Will these doc changes appear in v5.3.6?  They aren't currently showing in https://tiddlywiki.com/prerelease/

**Comment by Jermolene** [comment]
> Will these doc changes appear in v5.3.6? They aren't currently showing in https://tiddlywiki.com/prerelease/

No, I haven't merged this PR yet, but am planning to do so shortly.

**Comment by Jermolene** [comment]
Merged in 683aae4977b618e09572b74519468e131f99a11b

**Pull request closed without merging**